components/admin/requests/processing/TasksCard.tsx

```diff
@@ -254,11 +253,34 @@ export default function ProcessingTasksCard({ applicationId }: ProcessingTasksCa
               disabled={!reviewRequestCompleted}
```

Comment:
Let's also disable the button while the invoice generation is loading to prevent firing multiple times

Reply:
Ah good point, this is for uploadDocuments button tho. I added in the invoice generation button